regolith/helpers/a_manurevhelper.py

```diff
@@ -0,0 +1,122 @@
+"""Builder for manuscript reviews."""
+import datetime as dt
+import sys
+import time
+from argparse import RawTextHelpFormatter
+
+import nameparser
+
+from regolith.helpers.basehelper import SoutHelperBase, DbHelperBase
+from regolith.dates import month_to_int, month_to_str_int
+from regolith.fsclient import _id_key
+from regolith.sorters import position_key
+from regolith.tools import (
+    all_docs_from_collection,
+    filter_grants,
+    fuzzy_retrieval,
+)
+
+ALLOWED_STATI = ["invited", "accepted", "declined", "downloaded", "inprogress",
+                 "submitted", "cancelled"]
+
+
+def subparser(subpi):
+    subpi.add_argument("name", help="Full name of the first author",
+                       )
+    subpi.add_argument("due_date", help="due date in form YYYY-MM-DD in quotes", default=''
+                       )
+    subpi.add_argument("-d", "--database",
+                       help="The database that will be updated. Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("-q", "--requester",
+                       help="Name of the Program officer requesting"
+                       )
+    subpi.add_argument("-r", "--reviewer",
+                       help="name of the reviewer. Defaults to sbillinge"
```

Comment:
this should default to the id in config.json (see Jiawei for details), not sbillinge

Reply:
Not completed yet. This is what I am having trouble with since Jiawei seems to be busy and unable to answer my emails.

Reply:
maybe dig around in the closed PRs to find hers (I forget which it was) and then look in there if you can't reach her.

Reply:
here it is:
```
        if not rc.id:
            try:
                rc.id = rc.default_user_id
            except AttributeError:
                print(
                    "Please set default_user_id in '~/.config/regolith/user.json',\
                     or you need to enter your group id in the command line")
                return
```
you can search for exactly where it is used....in the  a_todo helper